gamebox/Models/ExternalGame.cs

```
@@ -0,0 +1,36 @@
+﻿namespace GameBox.Models
+{
+    /// <summary>
+    /// Holds information for a game
+    /// </summary>
+    public class ExternalGame
+    {
+        /// <summary>
+        /// ID of this game from the source
+        /// </summary>
+        public int? ExternalID { get; set; }
+        /// <summary>
+        /// List of Platforms this game is available on
+        /// </summary>
+        public List<ExternalPlatform>? Platforms { get; set; }
+        /// <summary>
+        /// Title of the game
+        /// </summary>
+        public string? Title {  get; set; }
```

I'm assuming that the `?` means that this is a nullable field? It's probably best if it wasn't nullable and the API should return an error if it's trying to put a nullable entry into here. We definitely expect (and want) there to be a title for every game.

The only field in this model that I would expect to be nullable is `ImagePath`, in case there is no image.